src/stylesheets/core/_variables.scss

```diff
@@ -1211,8 +1211,9 @@ $nav-width:                     951px !default;
 $usa-btn-small-width:           units(6);
 $search-min-width:              27ch; // minimum length of 27 letters for typical search query
 
-// 44 x 44 pixels hit target following Apple iOS Human Interface Guidelines
-$hit-area: 4.4rem !default;
+// Was 44 x 44 pixels hit target following Apple iOS HIG
+// Now 40px to conform with standard units — is this a problem?
```

Comment:
Yeah it's a problem bc it's out of conformance with WCAG, 44px is the minimum we should go https://www.w3.org/WAI/WCAG21/Understanding/target-size.html: why not go a step above?